tests/conftest.py

```diff
@@ -102,7 +103,7 @@ def _logged_in_client(_test_client):
             "/auth/login",
             json={
                 "email_address": "test@fairhub.io",
-                "password": "Testingyeshello11!",
+                "password": "asdkj45@ksdSA",
```

Comment:
We should revert this the testing password. The mechanism for testing will be updated in the future to use a different db